elide-core/src/main/java/com/yahoo/elide/core/dictionary/EntityDictionary.java

```diff
@@ -189,6 +189,23 @@ public EntityDictionary(Map<String, Class<? extends Check>> checks, Injector inj
         this(checks, injector, Collections.emptySet());
     }
 
+    /**
+     * Instantiate a new EntityDictionary with the provided set of checks and an injection function.
+     * In addition all of the checks * in {@link com.yahoo.elide.core.security.checks.prefab} are mapped
+     * to {@code Prefab.CONTAINER.CHECK} * (e.g. {@code @ReadPermission(expression="Prefab.Role.All")}
+     * or {@code @ReadPermission(expression="Prefab.Common.UpdateOnCreate")})
+     * @param checks a map that links the identifiers used in the permission expression strings
+     *               to their implementing classes
+     * @param roleChecks a map that links the user check identifiers used in the permission expression strings
+     *      *               to their User Check object
```

Comment:
A few typeos:
1. Look like there are asterisks not starting new lines.
2. Change "In addition all of the checks" to "In addition, all of the checks".